.github/workflows/percy_snapshots.yml

```diff
@@ -55,3 +55,4 @@ jobs:
           PERCY_TOKEN: ${{ secrets.PERCY_TOKEN }}
           PERCY_PARALLEL_NONCE: ${{ inputs.PERCY_PARALLEL_NONCE }}
           PERCY_PARALLEL_TOTAL: ${{ inputs.PERCY_PARALLEL_TOTAL }}
+          REACT_APP_MAPBOX_API_KEY: ${{ secrets.REACT_APP_MAPBOX_API_KEY }}
```

Comment:
I believe you also need to update the inputs section above to accept a `REACT_APP_MAPBOX_API_KEY` secret input.

For testing, you could update component lib's github action to point to this pr's branch and push the commit to see if it runs as expected using your new changes.